Userland/Applications/FileManager/main.cpp

```diff
@@ -808,13 +808,18 @@ ErrorOr<int> run_in_windowed_mode(DeprecatedString const& initial_location, Depr
 
     auto copy_path_action = GUI::Action::create(
         "Copy Path", [&](GUI::Action const&) {
-            auto paths = directory_view->selected_file_paths();
-            if (paths.is_empty()) {
-                paths = tree_view_selected_file_paths();
+            Vector<DeprecatedString> selected_paths;
+            if (directory_view->active_widget()->is_focused()) {
+                selected_paths = directory_view->selected_file_paths();
+            } else if (tree_view.is_focused()) {
+                selected_paths = tree_view_selected_file_paths();
             }
-            VERIFY(!paths.is_empty());
+            VERIFY(!selected_paths.is_empty());
 
-            GUI::Clipboard::the().set_plain_text(paths[0]);
+            StringBuilder joined_paths_builder;
+            // Chosing ":" as delimiter because it can be easily transformed to different characters
+            joined_paths_builder.join(":"sv, selected_paths.span());
```

Comment:
1. A colon (":") is a valid filename character. Actually, every character except '/' is a valid filename character, which means that _all_ characters are valid path characters. I'd argue that newlines are less common in filenames than colons, so you can use that.
2. Instead of a StringView, you can pass a `char` here (`.join('\n', ...)`)

Reply:
Oh, I didn't know about it.

Reply:
> every character except '/' is a valid filename character

Then why `\n` makes more sense to you? I am curious.

Reply:
Done!